lib/kinetic_cafe/error.rb

```diff
@@ -176,6 +176,10 @@ def default_status
       defined?(Rack::Utils) && :bad_request || 400
     end
 
+    def default_level
+      ::Logger::Severity::ERROR
```

Comment:
Made some updates, hadn't even seen this. I ended up rethinking it and coming to about the same conclusion. See new diff for implementation (using `Logger#add`, since `Rails.logger` is just an instance of the standard Ruby `Logger` from what I've been able to tell.